packages/block/Block.tsx

```diff
@@ -1,22 +1,32 @@
-import React, { ForwardedRef, forwardRef } from 'react'
-import { BlockStyle } from './BlockStyles'
-import { BlockProps } from './types'
+import { ForwardedRef, forwardRef } from 'react'
+import { BlockStyle, BlockStyleProps } from './BlockStyles'
+import { BlockColors, BlockVariants } from './constants'
 
-function Block(props: BlockProps, ref?: ForwardedRef<HTMLDivElement>) {
-  const {
+export type BlockProps = Omit<
+  BlockStyleProps,
+  '$color' | '$variant' | '$paddingLess'
```

Comment:
```suggestion
  `$${string}`
```

Reply:
What is string variable?

Reply:
Discussed in DM, it's a bit painful to list these props manually, but it's unsafe from type perspective to omit all strings, which starts with $